<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database</class>
 <widget class="QDialog" name="Database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_savename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>READ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
